main/reference/ertp-api/amount-math.md

```diff
@@ -25,22 +25,19 @@ not equal, an error is thrown and no changes are made.
 Creates an **Amount** from a given **Brand** and **AmountValue**.
 
 ```js
-// amount837 = { brand: quatloosBrand, value: 837n }
-const amount837 = AmountMath.make(quatloosBrand, 837n);
+const bid = AmountMath.make(quatloosBrand, 300n);
 ```
 
 ## AmountMath.coerce(brand, allegedAmount)
 - **brand**: **[Brand](./brand.md)**
 - **allegedAmount**: **[Amount](./ertp-data-types.md#amount)**
 - Returns: **Amount**
 
-Makes sure an **Amount** is for the specified *brand* and if so, returns it.
-If the **Amount** is for an invalid **Brand**, this method throws an exception.
+Verifies that an **Amount** is for the specified *brand* and returns a new equivalent **Amount**.
```

Comment:
```suggestion
Verifies that an **Amount** is for the specified *brand* and returns an equivalent **Amount**.
```

The PR overcorrects the original. The original mistakenly implied it is the same amount object. The PR mistakenly states it is guaranteed to be a fresh amount object, even if the original would have been fine. I think the API spec should only guarantee equivalence, without stating whether or not it is a new JS allocation.